core/sql/exp/exp_conv.cpp

```diff
@@ -9321,6 +9321,32 @@ convDoIt(char * source,
   };
   break;
 
+// gb2312 -> utf8
+// JIRA 1720
+  case CONV_GBK_F_UTF8_V:
+  {
+    int copyLen = 0;
+    int convLen = gbkToUtf8( source, sourceLen, target, targetLen);
+    if (convLen > 0) {
```

Comment:
yes, this is correct, thanks a lot, if the source is empty string, the target will be empty string as well and the convLen will be 0. So it is a valid case.